### PR TITLE
Talk descriptions

### PR DESCRIPTION
**Pull request by olyavertwist**
I've added descriptions for all talk-api endpoints except for those that are private. Additionally, I've assigned "x-access: private" on the method level (for documentation generation purposes).

With the next PR, I'll update components and their descriptions.

### CODE REVIEW

File: talk/talk-api.yaml [hunk 1]
```diff
@@ -2134,24 +2184,52 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ResponseError'
+  /webhooks/phoneNumbers/port:
+    x-access: private
```
Comment (BullDog57Rus) [inline talk/talk-api.yaml:2188]: as webhooks are private, let's leave them near their original family :)

**Review by BullDog57Rus — CHANGES_REQUESTED** [review]

File: talk/talk-api.yaml (hunk 1 above)
Comment (olyavertwist) [reply talk/talk-api.yaml]: For some reason, we had 2 same /webhooks/phoneNumbers/port endpoints. I eliminated one of them and moved the other to its logical "parent" /phoneNumbers.

File: talk/talk-api.yaml [hunk 2]
```diff
@@ -591,19 +599,19 @@ paths:
...
-      summary: Partially update a voice call
+      summary: Partially update a call
+      description: Use this endpoint to partially modify a call without overwritting it. Provide an ID in the path.
       operationId: patchCallById
       parameters:
         - $ref: '#/components/parameters/ResourceId'
       requestBody:
-        description: Call to update
-        content:
+        description: In the request body, pass the updated CallChangeRequest.
```
Comment (BullDog57Rus) [inline talk/talk-api.yaml:608]:
```suggestion
        content:
        description: In the request body, pass the updated CallChangeRequest.
```

File: talk/talk-api.yaml [hunk 3]
```diff
@@ -591,19 +599,20 @@ paths:
...
-      summary: Partially update a voice call
+      summary: Partially update a call
+      description: Use this endpoint to partially modify a call without overwritting it. Provide an ID in the path.
       operationId: patchCallById
       parameters:
         - $ref: '#/components/parameters/ResourceId'
       requestBody:
-        description: Call to update
         content:
+        description: In the request body, pass the updated CallChangeRequest.
```
Comment (BullDog57Rus) [inline talk/talk-api.yaml:609]:
```suggestion
        description: In the request body, pass the updated CallChangeRequest.
        content:
```

**Review by BullDog57Rus — APPROVED** [review]

**Pull request merged**